app/views/entities/new.html.erb

```diff
@@ -0,0 +1,25 @@
+<header>
+    <div>
+        <%= link_to image_tag('back.png'), group_entities_path(group_id: @group.id), id: "back-icon" %>
+    </div>
+    <h2>
+        Transaction
+    </h2>
+    <div>
+        <%= button_to "Logout", destroy_user_session_path, method: :delete %>
+    </div>
+</header>
+
+<%= form_for(Entity.new, url: group_entities_path(group_id: @group.id)) do |f| %>
+    <div class="field tform mb-2 mt-4">
+        <%= f.text_field :name, placeholder: "Name", autofocus: true, autocomplete: "name", class: "form-control form-control-lg" %>
+    </div>
+
+    <div class="field tform mb-2">
+        <%= f.number_field :amount, placeholder:"Amount", autofocus: true, autocomplete: "name", class: "form-control form-control-lg" %>
+    </div>
+
+    <div class="actions">
+        <%= f.submit "Add Transaction", class: "btn btn-primary btn-lg btn-block splashBtn" %>
+    </div>
+<% end %>
```

Comment:
- [OPTIONAL] The requirements state that the form page for adding a new _**transaction/record**_ should have a form input where the user can choose at least one existing category that the new record/transaction will belong to. Kindly implement this project requirement.